docs/release-notes/release-notes-0.8.2.md

```diff
@@ -0,0 +1,129 @@
+
+The fix in [this PR](https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/822) removes the method of essentially "watermarking" the taker's inputs with a random set of bytes `0xdeadbeef` and instead just checks which inputs belong to the taker; this removal is thus also a code cleanup, but mainly, it removes the need to send a *distinct* copy of the transaction to the makers, without watermarks, while keeping a watermarked copy locally (this was the situation before the introduction of the bug in July 2020).
+
+`a340f06` remove magic 'deadbeef' identifier from taker.py
+
+### Adds SNICKER for testing
+
+[This PR](https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/768) is now merged, and adds several tools for Joinmarket users to co-sign (as "receiver") or also create and publish (as "proposer") SNICKER-style coinjoins. For more information on what this is, read the [documentation](https://github.com/JoinMarket-Org/joinmarket-clientserver/blob/master/docs/SNICKER.md). Notice that this is by default switched off. Yieldgen bots can have this service running in the background to receive SNICKER coinjoins, but that specific function is explicitly disabled for mainnet, for now; use of signet is recommended for those wishing to experiment, but if you did wish to try this out on mainnet, it's better to use the manual receiver script.
+
+Also, those not interested in *this* function might still be interested in the script `scripts/snicker/snicker-finder.py` which can be used to scan for Joinmarket coinjoins with the `-j` flag.
+
+`53a1822` Adds SNICKER functionality
+
+`a857c12` disable for mainnet yieldgens
+
+### Payjoin bugfixes
+
+While other refactoring work is going on for payjoin, the basic functionality was discovered to have a couple of bugs, one very minor but the other quite meaningful.
+The minor one was in Qt: the payjoin code was trying to broadcast the fallback after the success of the payjoin transaction, which results only in misleading log messages, but this was fixed.
+The bigger one: if receiver's payjoin transaction used multiple of its own inputs, there was a bug in the sanity checks carried out by the sender that resulted in these proposals being rejected, and falling back to the non-payjoin payment; this has been fixed.
+
+`65b06ef` Fix Qt BIP78receiver trying fallback after success
+
+`dc4a5b2` Bugfix: correct check of receiver inputs in BIP78
+
+### Improvements to ob-watcher function
+
+The main point here is explained in [this PR](https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/803) which addresses issue #804. Local hosting creates the risk of MitM attacks and privacy leaks (e.g. leaking the IP address of a user of Joinmarket).
```

Comment:
```suggestion
The main point here is explained in [this PR](https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/803) which addresses issue #804. Local hosting removes the risk of MitM attacks and privacy leaks (e.g. leaking the IP address of a user of Joinmarket).
```

Reply:
doh, thanks

Reply:
Fixed in https://github.com/JoinMarket-Org/joinmarket-clientserver/commit/d235638b715ac8d67d8e9ef077241ec9df95ef55